src/components/staking/LiValidator.vue

```diff
@@ -38,6 +38,9 @@
         <h3 class="li-validator-name">
           {{ validator.name }}
         </h3>
+        <h3 class="li-validator-name li-validator-name-short">
```

Comment:
we should use CSS to achieve this, not extra HTML. also, this is not related to livepeer so could be addressed in a separate PR. 

livepeer validator names are temporarily addresses. this is not going to be the case for long and is not a precedent we want to set.

Reply:
Ok, you are right. I just deleted the whole logic for `validator.name` from `LiValidator`. It is better to invest time adding those names to the DB than implementing this for a short period of time only

Reply:
Ok, you are right. I just deleted the whole logic for `validator.name` from LiValidator. It is better to invest time adding those names to the DB than implementing this for a short period of time only